internal/characters/razor/cons.go

```diff
@@ -97,7 +97,7 @@ func (c *char) c6() {
 		}
 		c.Core.QueueAttack(
 			ai,
-			combat.NewCircleHit(c.Core.Combat.Player(), 0.5),
+			combat.NewCircleHit(c.Core.Combat.Player(), 1.5),
```

Comment:
I forgot to mention that you should pull out the target out of the OnEnemyDamage event here just like you did for Yanfei A4